### ISSUE TITLE
Running Oracledb on concourse CI/CD pipeline is showing error

### ISSUE THREAD

**Issue by Tash2610** [oracle/node-oracledb#1548]
1. What versions are you using?
5.5.0

2. Describe the problem.
While running playwright typescript automation project with Oracledb as dependency on concourse pipeline, pipeline is getting errored out due to  ERROR: Unable to connect to database: Error: DPI-1047: Cannot locate a 64-bit Oracle Client library: "libclntsh.so: cannot open shared object file: No such file or directory"

3. Are you installing into vanilla Node.js, or using something like Electron?
Using dockerfile to create an image of automation project and then run the image on the concourse CI/CD pipeline.
4. Show the directory listing where your Oracle Client libraries are installed (e.g. the Instant Client directory).  Is it 64-bit or 32-bit?

5. Show what the `PATH` environment variable (on Windows) or `LD_LIBRARY_PATH` (on Linux) are set to?

6. Show any Oracle environment variables set (e.g. ORACLE_HOME, ORACLE_BASE).

**Comment by Tash2610** [comment]
@cjbj is there a way to handle the oracledb library missing error through package.json or dockerfile?

**Comment by cjbj** [comment]
See the blog post series link I gave earlier which shows how to do it in the Dockerfile.

**Comment by Tash2610** [comment]
@cjbj Thank you for sharing those insightful links. 
We already have a dockerfile in our playwright automation project, as added below, where we're already using the playwright image, and the steps which are mentioned in the [Docker for Oracle Database Applications in Node.js and Python](https://blogs.oracle.com/opal/post/part-1-docker-for-oracle-database-applications-in-nodejs-and-python) suggest to pull oracle client image first from Docker registry and then use that image in a dockerfile. As, we're already using Playwright image and can't use another image to create docker image, Can you suggest something to fix the CI/CD pipeline issue of missing OracleDB libraries through Dockerfile during pipeline execution on server? 
Dockerfile
```
FROM mcr.microsoft.com/playwright:v1.32.3-focal AS local
WORKDIR /app
COPY .npmrc .
COPY package.json .
RUN yarn install
COPY . .

FROM local AS ci
ARG ACCOUNT_ENV
ENV ACCOUNT_ENV=$ACCOUNT_ENV
```

**Comment by cjbj** [comment]
The blog post shows various ways to install and configure Instant Client. Check it out!

**Comment by cjbj** [comment]
Closing this since the solutions for client library installation and loading have been shared.

The [just-released node-oracledb 6.0](https://medium.com/@sharad-chandran/usher-in-a-new-era-with-the-node-oracledb-6-0-pure-javascript-thin-driver-e10e2af693b2) doesn't necessarily need Instant Client libraries so there is also an easier solution now.
